### PR TITLE
Update README to mention plugins

### PR DESCRIPTION
**Pull request by MaxLeiter**
### Short Description:

Plugins are now mentioned and listed in the README, --mode replacement is explained, and I added a title to a weird section of the FAQ
### Changes:
- Added section to table of contents
- Added a 'Where's `--mode`/`-m`?' section to the FAQ
- Added a 'Plugins' section

@OpenPoGo/maintainers

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -143,9 +145,10 @@ The status code "3" corresponds to "Flee" - meaning your Pokemon has ran away.
    {"responses": { "CATCH_POKEMON": { "status": 3 } }
 ### Why aren't my pokemon showing up in my Pokedex?
 Finish the tutorial on a smartphone. This will then allow everything to be visible.
+### Where's `--mode`/`-m`?
+Now, instead of using `--mode` you need to exclude plugins. If you don't want to catch pokemon, exclude the `catch_pokemon` plugin  (`-ep catch_pokemon`), and if you don't want to farm pokestops just exclude the `farm_pokestop` plugin (`-ep spin_pokestop`). Alternatively, you can modify your configuration JSON file to do the same thing.
```
Comment (wchill) [inline README.md]: > `farm_pokestop`

Should be `spin_pokestop`

**Comment by MaxLeiter** [comment]
Should be fixed, @wchill

**Comment by andythorne** [comment]
:+1: 

[Approved with PullApprove](https://pullapprove.com/OpenPoGo/OpenPoGoBot/pull-request/166/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=OpenPoGo/OpenPoGoBot)

**Comment by Reaver01** [comment]
👍 

[Approved with PullApprove](https://pullapprove.com/OpenPoGo/OpenPoGoBot/pull-request/166/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=OpenPoGo/OpenPoGoBot)

**Pull request merged**